.github/workflows/binaries.yml

```diff
@@ -0,0 +1,51 @@
+name: Test
```

Comment:
Looks good, but why the `name` is `Test`?

Reply:
Whoops, renamed!